espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ViewFinder.kt

```diff
@@ -189,6 +191,14 @@ object ViewFinder {
                     emptyList()
                 }
             }
+            Strategy.VIEWMATCHER -> {
+                val matcherJson = ViewMatcherJson.fromJson(selector)
+                views = try{
```

Comment:
Please apply formatting to the file (we usually put space after try/before catch)